src/app.rs

```diff
@@ -41,6 +41,9 @@ impl<B: Backend> App<B> {
             config.marker_topics,
             config.marker_array_topics,
             config.map_topics,
+            config.pose_stamped_topics,
```

Comment:
sort!

Reply:
Alphabetically? Alright!

Reply:
Wait should I sort topics everywhere? Because they are definitely NOT sorted